backend/sources/whatsapp/connector/src/main/java/co/airy/core/sources/whatsapp/api/MetaConfig.java

```diff
@@ -0,0 +1,19 @@
+package co.airy.core.sources.whatsapp.api;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class MetaConfig {
+    @Bean
+    @Qualifier("metaObjectMapper")
```

Comment:
So far the meta-object mapper has been the same as ours in terms of config. It's better however to explicitly split them.